vue-components/src/components/TextInput.vue

```diff
@@ -64,6 +65,13 @@ export default Vue.extend( {
 		},
 	},
 
+	data() {
+		return {
+			// https://github.com/vuejs/vue/issues/5886
+			id: `wikit-TextInput-${Math.floor( Math.random() * 1000000 )}`,
```

Comment:
As I already foresee us generating an Id for the lookup input as well (and potentially radios and checkboxes in the future), maybe we should abstract this into a helper?

Reply:
Yup, makes perfect sense to me given we already want to use it in the Lookup component! Done!